tests/x11/evolution/evolution_prepare_servers.pm

```diff
@@ -33,6 +33,10 @@ sub run() {
     }
     else {
         zypper_call("in dovecot", exitcode => [0, 102, 103]);
+	if (is_opensuse) {
+	    zypper_call("in --force-resolution postfix", exitcode => [0, 102, 103]);
```

Comment:
Why? That should not be necessary unless you hit a bug. In that case please use `record_soft_failure` with a reference to the bug when applying the workaround.

Reply:
because for some reason on TW exim is installed and this is very weird because using the default TW image I can see that postfix is installed. And I couldn't find any test that is explicitly installing exim, so I had to use the force (cit.) here.

I can't avoid that step though, otherwise postfix will not be installed. 

Ok for adding `rsf` :+1:

Reply:
@okurz in such a case do I have to open a bug in poo or in boo?